x/evm/types/state_transition.go

```diff
@@ -57,7 +57,7 @@ func GetHashFn(ctx sdk.Context, csdb *CommitStateDB) vm.GetHashFunc {
 		case ctx.BlockHeight() == int64(height):
 			// Case 1: The requested height matches the one from the context so we can retrieve the header
 			// hash directly from the context.
-			return HashFromContext(ctx)
+			return csdb.bhash
```

Comment:
update comment

Reply:
@zhongqiuwood @summerpro  can we get a quick update so that we can merge? sorry for the late response!

Reply:
updated